### ISSUE TITLE
Problem of getting started (Starting and launching the algorithm and the MedRecUI)

### ISSUE THREAD

**Issue by gmbinf** [jherskovic/MedRec#7]
Hello Doctor,
I am a student at the University of Medicine in Paris Descartes (France) and I am currently working on a research tool on medication reconciliation.

I downloaded from there all files, as well as RXNOM and all other codes related to medication reconciliation (MedRecUI, gwt-dnd, gwt-fx).

my problem is how can i use your librarys/classes & all your data ? 
which environnement should i use ? 

to summarize, i need a getting started/wiki ...

i'm looking forward to your fedback 
I thank you in advance
kind Regards;
